r/vignettes/install.Rmd

```diff
@@ -7,6 +7,12 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+TL;DR
+
+In most cases, `install.packages("arrow")` should just work. There are things you can do to make the installation faster (see below). If, for any reason, it doesn't work, set the environment variable `ARROW_R_DEV=true` in your `.Renviron`, retry, and share the logs with us.
```

Comment:
I think we should remove the reference to `.Renviron` here, as to me it implies this is the only way to do, this rather than just one of the ways.

Reply:
Done